projects/kit/components/multi-select/multi-select.template.html

```diff
@@ -41,14 +41,14 @@
             [context]="context"
             [content]="valueContent"
         ></div>
-        <tui-svg
-            *ngIf="interactive"
+
+        <div
+            polymorpheus-outlet
             automation-id="tui-multi-select__arrow"
             tuiPreventDefault="mousedown"
             class="t-arrow"
-            [class.t-arrow_open]="open"
```

Comment:
Take a look at LESS file, probably some styles can be removed if we rely on `TUI_ARROW` now.

Reply:
fixed